### PR TITLE
Exercise 10

### PR DESCRIPTION
**Pull request by Cathe313**

### CODE REVIEW

File: exercise-10.txt [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+Query OK, 0 rows affected (0.07 sec)
+Records: 0  Duplicates: 0  Warnings: 0
+
+mysql> DESCRIBE Floors;
++--------------------+---------+------+-----+---------+----------------+
+| Field              | Type    | Null | Key | Default | Extra          |
++--------------------+---------+------+-----+---------+----------------+
+| id                 | int(11) | NO   | PRI | NULL    | auto_increment |
+| floorNb            | int(11) | YES  |     | NULL    |                |
+| nbRooms            | int(11) | YES  |     | NULL    |                |
```
Comment (nyanofthemoon) [inline exercise-10.txt]: You could omit this column since you have Rooms.floorId :)

Comment (Cathe313) [inline exercise-10.txt]: True :) I'll change that.

File: exercise-10.txt [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+mysql> DESCRIBE Availability;
++-------------------------------------+----------+------+-----+---------+----------------+
+| Field                               | Type     | Null | Key | Default | Extra          |
++-------------------------------------+----------+------+-----+---------+----------------+
+| id                                  | int(11)  | NO   | PRI | NULL    | auto_increment |
+| date                                | datetime | YES  |     | NULL    |                |
+| roomOccupancySingleAvailable        | int(11)  | YES  |     | NULL    |                |
+| roomOccupancyDoubleAvailable        | int(11)  | YES  |     | NULL    |                |
+| roomOccupancyTripleAndMoreAvailable | int(11)  | YES  |     | NULL    |                |
```
Comment (nyanofthemoon) [inline exercise-10.txt]: How would you prevent Room from being booked for both Single and Double occupancy?

Comment (Cathe313) [inline exercise-10.txt]: I could either collapse `roomOccupancySingleAvailable` and `roomOccupancyDoubleAvailable` into a single column, or create another column that encompasses both.

**Pull request closed without merging**